.github/workflows/integration-tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
Can we use [`exclude`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#excluding-matrix-configurations) to exclude 3.13 runs from combining with older versions of F´ that break on it - rather than not running 3.13 at all ?

Reply:
```
    exclude:
      - python-version: 3.13
        fprime-version: 3.8
[...]
```